install.sh

```diff
@@ -2691,6 +2691,7 @@ If you find it useful, please show your support by starring the project in GitHu
 #   0 when successfully finished, 1 if failed
 ############################################################
 execute_installation() {
+    TARGET_TEMPLATE_DIR=""
```

Comment:
Nit if you don't want to bother, but I'd move this to the start of `parse_command_line_arguments`.